content/en/videos/video15.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Improve Vulnerability Management with OCI Artifacts -It Is That Easy!"
```

Comment:
To simplify this PR review process, I would suggest keeping this video only. Other videos can be updated by other maintainers.